.ci/jenkins/Jenkinsfile.deploy

```diff
@@ -371,6 +371,10 @@ boolean isDeployLatestTag() {
     return params.DEPLOY_WITH_LATEST_TAG
 }
 
+boolean isDeployImage() {
+    return params.DEPLOY_IMAGE ? params.DEPLOY_IMAGE.toBoolean() : false
```

Comment:
you need to define this parameter into the `setupDeployJob` method in `jobs.groovy` file

Reply:
But this parameter is already set in BuildImage

Reply:
setupBuildImageJob -> Jenkinsfile.build-image
setupDeployJob -> Jenkinsfile.deploy

Reply:
Doesn't deploy call the build image?

Reply:
it does but here you are trying to access a parameter which is not defined for the deploy job ... you need to define it

Reply:
@radtriste done.